cmd/manager/main.go

```diff
@@ -84,8 +84,7 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Metrics:                server.Options{BindAddress: metricsAddr},
```

Comment:
Nice to have for performance evaluations would be to add a flag that sets the pprof endpoint to enable/disable the default pprof implementation through controller-runtime. Could be done in a follow-up though and shouldn't hold this PR